Source/SvgElementIdManager.cs

```diff
@@ -153,15 +153,6 @@ public string EnsureValidId(string id, bool autoForceUniqueID = false)
                 return id;
             }
 
-            if (char.IsDigit(id[0]))
-            {
-                if (autoForceUniqueID)
-                {
-                    return EnsureValidId("id" + id, true);

```

Comment:
Maybe leave this part in...

Reply:
@mrbean-bremen 

Thank you for your review.

ID is converted as follows:
`41cb5de8-82e7-4ab5-94d7-87e10775c848` -> `id41cb5de8-82e7-4ab5-94d7-87e10775c848`

However, other unusable characters (. and -) are not converted, so there is no this issue.

So I think this conversion is unnecessary.

Reply:
Ok, thanks!